docs/cxx-proposal/bridges.md

```diff
@@ -0,0 +1,37 @@
+# Proposed Language Bridges
+
+## Python
+
+Since OTIO originated as Python (and has an extensive test suite, in Python), our starting position
+is that existing Python code (adapters, plugins, schemadefs) must continue to work, as currently
+written. Python code in the `core` or `schema` directories will of course be rewritten, but Python code
+outside those modules should not be aware of any change.
```

Comment:
We have already made a few pre-emptive changes to the Python API to ease the transition to C++, so this statement is stronger than we really plan. I don't want to soften this too much, but we should be honest by noting that *some* changes are happening, but only when necessary or desirable (e.g. the immutability of TimeRange, etc.) and that we are doing this ahead of time to give people time to adjust.